spec/requests/internal/users_spec.rb

```diff
@@ -39,7 +39,7 @@
     end
   end
 
-  describe "PUT internal/users/:id/edit" do
+  describe "POST internal/users/:id/banish" do
```

Comment:
Just noticed that the description of this `describe` didn't  fit the specs, probably a copy/paste artifact.